e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -103,6 +103,7 @@ describeIfIAMV2p1('projects API', () => {
     const noRulesStr = 'NO_RULES';
     const editsPendingStr = 'EDITS_PENDING';
     const rulesAppliedStr = 'RULES_APPLIED';
+    const numberOfNodesAdded = 4;
```

Comment:
maybe a comment that it must correspond to the number of fixtures loaded L151-155?